src/types/kubectl.ts

```diff
@@ -98,6 +98,14 @@ export class Kubectl {
       return await this.execute(args)
    }
 
+   public async annotateFile(
```

Comment:
is this needed vs annotate files? AnnotateFiles works with a single file passed in

Reply:
didn't realize it would work with a single one, but yeah if that's the case then no there shouldn't be any need